frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/header/field_cell.dart

```diff
@@ -172,7 +172,7 @@ class FieldCellButton extends StatelessWidget {
       leftIcon: FlowySvg(
         name: field.fieldType.iconName(),
       ),
-      radius: BorderRadius.zero,
+      radius: BorderRadius.circular(6),
```

Comment:
Hello @hyj1204, the improvements look great except for one small thing I've noticed. This change does make the row details page look better, but this is also the same widget used in the headers, so they look like this now: 

![image](https://user-images.githubusercontent.com/71320345/231117729-30f19861-0e71-4a79-97d4-32a58aa9069c.png)

Could you patch the corners up?